Content.Shared/CCVar/CCVars.cs

```diff
@@ -1209,24 +1209,12 @@ public sealed class CCVars : CVars
         /// </summary>
         public static readonly CVarDef<bool> WhitelistEnabled =
             CVarDef.Create("whitelist.enabled", false, CVar.SERVERONLY);
-
-        /// <summary>
-        ///     The loc string to display as a disconnect reason when someone is not whitelisted.
-        /// </summary>
-        public static readonly CVarDef<string> WhitelistReason =
-            CVarDef.Create("whitelist.reason", "whitelist-not-whitelisted", CVar.SERVERONLY);
-
-        /// <summary>
-        ///     If the playercount is below this number, the whitelist will not apply.
-        /// </summary>
-        public static readonly CVarDef<int> WhitelistMinPlayers =
-            CVarDef.Create("whitelist.min_players", 0, CVar.SERVERONLY);
-
         /// <summary>
-        ///     If the playercount is above this number, the whitelist will not apply.
+        ///     Specifies the whitelist prototypes to be used by the server. This should be a comma-separated list of prototypes.
+        ///     If a whitelists conditions to be active fail (for example player count), the next whitelist will be used instead. If no whitelist is valid, the player will be allowed to connect.
         /// </summary>
-        public static readonly CVarDef<int> WhitelistMaxPlayers =
-            CVarDef.Create("whitelist.max_players", int.MaxValue, CVar.SERVERONLY);
+        public static readonly CVarDef<string> WhitelistPrototype =
+            CVarDef.Create("whitelist.prototypeList", "basicWhitelist", CVar.SERVERONLY);
```

Comment:
```suggestion
            CVarDef.Create("whitelist.prototype_list", "basicWhitelist", CVar.SERVERONLY);
```

CVar naming convention is pascal case